code/game/machinery/doors/poddoor.dm

```diff
@@ -274,6 +274,7 @@
 /obj/structure/machinery/door/poddoor/almayer
 	icon = 'icons/obj/structures/doors/blastdoors_shutters.dmi'
 	openspeed = 4 //shorter open animation.
+	var/bumpable = TRUE
```

Comment:
this variable isn't super clear, call it something like `vehicle_resistant` and then swap the current logic around

Reply:
var name has been changed from bumpable to vehicle_resistant